bindings/test/unit/test_server_info.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+import mock
+
+from pulp.bindings import server_info
+
+
+class TestServerstatus(unittest.TestCase):
+    def setUp(self):
+        self.server = mock.MagicMock()
+        self.api = server_info.ServerStatusAPI(self.server)
+
+    def test_correct_passthrough(self):
+        self.api.get_status()
+
+        self.server.GET.assert_called_once_with('v2/status/')
```

Comment:
Rather than making a setUp() and setting self.api and self.server there, it would be a little simpler to just do all that in this one method. If you prefer it the way it is, that's fine.